hr_applicant/models/hr_app_med_occ.py

```diff
@@ -0,0 +1,132 @@
+from datetime import datetime
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.translate import _
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+
+
```

Comment:
Remove extra blank lines

Reply:
yes mam removed

Reply:
removed extra blank lines and two space line between import and class